test/jdk/java/awt/Toolkit/DesktopProperties/rfe4758438.sh

```diff
@@ -28,20 +28,6 @@ OS=`uname`
 
 case "$OS" in
     Linux* )
-        GNOMESID=`pgrep gnome-session | head -n1`
```

Comment:
This was a hacky way to get environment variables, it may not work, depending on a system configuration.
It was was implemented before the https://github.com/openjdk/jtreg/pull/152, and is no longer needed, so removed.